publications-2019.bib

```diff
@@ -13,4 +13,20 @@ @Article{Basak2019a
   url       = {https://www.researchgate.net/profile/Anup_Basak/publication/326847352_Finite_element_procedure_and_simulations_for_a_multiphase_phase_field_approach_to_martensitic_phase_transformations_at_large_strains_and_with_interfacial_stresses/links/5baae99da6fdccd3cb733c04/Finite-element-procedure-and-simulations-for-a-multiphase-phase-field-approach-to-martensitic-phase-transformations-at-large-strains-and-with-interfacial-stresses.pdf},
 }
 
+@article{ZHANG2019634,
+  title = "A high-order and interface-preserving discontinuous Galerkin method for level-set reinitialization",
```

Comment:
For consistency, would you mind please changing the quotes used for all fields to braces? It helps protect against runaway arguments (something I had difficulty with when initially importing entries).